fixieai/agents/utils.py

```diff
@@ -125,42 +132,46 @@ def _validate_base_prompt(base_prompt: str):
 
 class FewshotLinePattern(enum.Enum):
     QUERY = re.compile(r"^Q:")
-    AGENT_SAYS = re.compile(r"^Agent\[\w+] says:")
-    FUNC_SAYS = re.compile(r"^Func\[\w+] says:")
-    ASK_AGENT = re.compile(r"^Ask Agent\[\w+]:")
-    ASK_FUNC = re.compile(r"^Ask Func\[\w+]:")
+    AGENT_SAYS = re.compile(r"^Agent\[(\w+)] says:")
```

Comment:
optional nit: consider giving these groups a name `(?P<agent_id>\w+)` to use below `match.group('agent_id')`

Reply:
That's nicer, thank you!